Notifier-service/src/services/streams/conferenceStream.ts

```diff
@@ -39,16 +43,72 @@ export class ConferenceStreamMongo extends ConferenceStream {
             })
         })
             .pipe(
-                share()
+                filter()
             )
+        this.insertstreamObs$ = Observable.create((observer: Observer<any>) => {
```

Comment:
* This can be done, but it will lead to creating extra chageStream listeners , instead of filtered out events. But that's fine. New stream for insert == less event callbacks.

* add share() operator. Read up on what it does. We want same data being propagated to all listeners from the same observable, at the same time. If u don't use share, new change stream event handllers will be creat